src/Paket.Core/ProjectFile.fs

```diff
@@ -1337,6 +1338,18 @@ type ProjectFile with
 
     member this.FindReferencesFile() = this.FindCorrespondingFile Constants.ReferencesFile
 
+    member this.FindLocalizedLanguageNames() =
+        let pattern = @"Properties\\Resources.(?<language>\w+(-\w+)?).resx"
+        let includeattribute = "Include"
+        this.ProjectNode
+        |> getDescendants "EmbeddedResource"
+        |> List.filter (hasAttribute includeattribute)
+        |> List.map (fun n -> n.Attributes.[includeattribute])
```

Comment:
maybe define tryGetAttribute

```fsharp
let tryGetAttributeValue name node = 
    if hasAttribute name node then
        Some node.Attributes.[name].Value
    else
        None
```

then you can replace List.filter/List.map/List.map with

```fsharp
|> List.choose (tryGetAttributeValue"Include")
```